packages/ncids-css/packages/nci-card/internal/_nci-card.scss

```diff
@@ -13,15 +13,19 @@
 		@include u-radius('md');
 		overflow: hidden;
 
+		/* Link Status Colors from USWDS
+		uswds/packages/uswds-core/src/styles/mixins/typography/typeset.scss */
 		&:visited .nci-card__title {
-			@include u-text($theme-link-visited-color, 'underline', 'bold');
+			color: color($theme-link-visited-color);
```

Comment:
For clarification, we're setting these colors specifically for the card's interactions, which is why they are copied from USWDS rather than simply using `typeset-link`.

These colors are pulled from USWDS, specifically the `typeset-link` mixin at `uswds/packages/uswds-core/src/styles/mixins/typography/typeset.scss`.

We omit the `focus-outline` present in USWDS from our card's focus interaction because the card itself is outlined on focus, and the focus on the title is decidedly unnecessary.